examples/dotNetClient/DotNetYuduAPIExample/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Globalization;
+using System.Security.Cryptography;
+using System.Text;
+using RestSharp;
+
+namespace DotNetYuduAPIExample
+{
+    internal class Program
+    {
+        private const string SecretKey = "Please contact Yudu to obtain this shared secret.";
+        private const string ApiKey = "Please contact Yudu to obtain this API key.";
```

Comment:
Clients don't necessarily need to contact Yudu for this - if they know where to find it when they login to Publisher or have already been given it by Yudu they can just enter the details.